### ISSUE TITLE
Not recognized

### ISSUE THREAD

**Issue by MatteoMeil** [begrossi/MediaScannerPlugin#6]
I installed this plugin in a ionic project but everytime I run a ionic command (i.e.: ionic cordova platform ecc.) it stops with an error saying this plugin was not found.
The reason is why this plugin is not copied in the `/plugins/` directory.

I also suggest you to mantain the same name for folders: in `node_modules` this plugin come as `cordova-plugin-mediascanner` while in `config.xml` is named as `br.brunogrossi.MediaScannerPlugin`

**Comment by jasvindersingh-evon** [comment]
Yes this is creating an issue. Everytime it has to be installed.
